content/source/docs/cloud/workspaces/index.html.md

```diff
@@ -75,3 +77,8 @@ We recommend that organizations break down large monolithic Terraform configurat
 For example, the code that manages your production environment's infrastructure could be split into a networking configuration, the main application's configuration, and a monitoring configuration. After splitting the code, you would create "networking-prod", "app1-prod", "monitoring-prod" workspaces, and assign separate teams to manage them.
 
 Much like splitting monolithic applications into smaller microservices, this enables teams to make changes in parallel. In addition, it makes it easier to re-use configurations to manage other environments of infrastructure ("app1-dev," etc.).
+
+## Creating Workspaces
+
+You can create workspaces through the [Terraform Cloud UI](/docs/cloud/workspaces/creating.html) or the [Workspaces API](../api/workspaces.html).
```

Comment:
Or the CLI, using the Cloud integration! With `tags` configured, you can use `terraform workspace new` and `terraform workspace delete`, etc.

Reply:
yes you're right!! let me add :-)